cmd/vclusterctl/cmd/snapshot/getrequest.go

```diff
@@ -0,0 +1,44 @@
+package snapshot
+
+import (
+	"github.com/loft-sh/log"
+	"github.com/loft-sh/vcluster/pkg/cli"
+	"github.com/loft-sh/vcluster/pkg/cli/completion"
+	"github.com/loft-sh/vcluster/pkg/cli/flags"
+	"github.com/loft-sh/vcluster/pkg/cli/util"
+	"github.com/spf13/cobra"
+)
+
+type GetRequestCmd struct {
+	*flags.GlobalFlags
+	Log log.Logger
+}
+
+func NewGetRequestCmd(globalFlags *flags.GlobalFlags) *cobra.Command {
+	getRequestCmd := &GetRequestCmd{
+		GlobalFlags: globalFlags,
+		Log:         log.GetInstance(),
+	}
+
+	_, nameValidator := util.NamedPositionalArgsValidator(true, false, "VCLUSTER_NAME")
```

Comment:
This hould be called "get" instead of "get-request"

Reply:
I have removed get-request and list-requests snapshot commands.